ludwig/backend/ray.py

```diff
@@ -794,13 +793,17 @@ def _prepare_batch(self, batch: pd.DataFrame) -> Dict[str, np.ndarray]:
 
 
 class RayBackend(RemoteTrainingMixin, Backend):
-    def __init__(self, processor=None, trainer=None, loader=None, use_legacy=False, **kwargs):
+    BACKEND_TYPE = "ray"
+
+    def __init__(self, processor=None, trainer=None, loader=None, use_legacy=False, data_preprocessor=None, **kwargs):
         super().__init__(dataset_manager=RayDatasetManager(self), **kwargs)
+        self._preprocessor_kwargs = data_preprocessor or {}
```

Comment:
Curious about the different naming (data_preprocessor vs preprocessor_kwargs). Are these required by upstream/downstream dependencies?

Reply:
I wanted to call them preprocessor_kwargs everywhere but I was worried that it was too similar to the processor argument and that's why I made them different. I don't mind making it preprocessor_kwargs everywhere if you don't think that will be confusing

Reply:
I updated the names